components/map/MapScreen.tsx

```diff
@@ -38,13 +39,13 @@ const MapScreen = () => {
       <Map onBottomSheetContentChange={handleBottomSheetChange} />
       <BottomSheet ref={bottomSheetRef} snapPoints={snapPoints}>
         <BottomSheetContent cn="bg-white">
-          <View className="bg-white">
-            <Field label={t('MapScreen.BottomSheet.title')}>
-              <TextInput />
-            </Field>
-            {bottomSheetContent ? (
-              <>
-                <Panel className="my-2 g-4">
+          <View className="bg-white g-3">
+            <View className="g-2">
+              <Field label={t('MapScreen.BottomSheet.title')}>
+                <TextInput />
+              </Field>
+              {bottomSheetContent && (
```

Comment:
Please prefer ternary operator for non-boolean values such as `bottomSheetContent`. It can lead to UI errors (displaying 0)

e.g. https://dev.to/aishanipach/dont-use-for-conditional-reasoning-3525

Reply:
okay will fix in the next PR